src/host/inputBuffer.cpp

```diff
@@ -905,76 +826,44 @@ bool InputBuffer::_CanCoalesce(const KeyEvent& a, const KeyEvent& b) const noexc
 // the buffer with updated values from an incoming event, instead of
 // storing the incoming event (which would make the original one
 // redundant/out of date with the most current state).
-bool InputBuffer::_CoalesceRepeatedKeyPressEvents(_Inout_ std::deque<std::unique_ptr<IInputEvent>>& inEvents)
+bool InputBuffer::_CoalesceEvent(const std::unique_ptr<IInputEvent>& inEvent) const noexcept

```

Comment:
How can this be const? It modifies `lastEvent` which is in `_storage`

Reply:
Because `std::unique_ptr` is a pointer and `const` in C++ has about as much meaning as me saying that I won't eat that second pudding today: https://godbolt.org/z/38334rzjc

Reply:
(The `const` will go away once I use `INPUT_RECORD`, because then there's no pointer anymore.)